cmd/loki/loki-docker-config.yaml

```diff
@@ -3,21 +3,16 @@ auth_enabled: false
 server:
   http_listen_port: 3100
 
-ingester:
-  lifecycler:
-    address: 127.0.0.1
-    ring:
-      kvstore:
-        store: inmemory
-      replication_factor: 1
-    final_sleep: 0s
-  chunk_idle_period: 1h       # Any chunk not receiving new logs in this time will be flushed
-  max_chunk_age: 1h           # All chunks will be flushed when they hit this age, default is 1h
-  chunk_target_size: 1048576  # Loki will attempt to build chunks up to 1.5MB, flushing first if chunk_idle_period or max_chunk_age is reached first
-  chunk_retain_period: 30s    # Must be greater than index read cache TTL if using an index cache (Default index read cache TTL is 5m)
-  max_transfer_retries: 0     # Chunk transfers disabled
-  wal:
-    dir: /loki/wal
+common:
+  path_prefix: /loki
+  storage:
+    filesystem:
+      directory: /loki/chunks
```

Comment:
this directory is now only being used for chunk storage config, so I wonder if this property name still makes sense? should we change the filesystem storage type to allow a definition of `chunks_directory` and `rules_directory` instead of just `directory`?